ignite/trainer.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import division
+
+import time
+from collections import Iterable
+from enum import Enum
+
+from torch.autograd import Variable
+
+from ignite.history import History
+from ignite.engine import Engine
+from ignite._utils import _to_hours_mins_secs
+
+__all__ = ["TrainingEvents", "Trainer", "create_supervised"]
+
+
+class TrainingEvents(Enum):
```

Comment:
IMO, calling TrainingEvents.TRAINING_EPOCH_STARTED seems a little redundant.

What do you think about `TrainingEvents.EPOCH_STARTED`

merging `EvaluationEvents` with `TrainingEvents` in a more general event holder `Events` so it could be use as `Events.TRAINING_EPOCH_STARTED`?

Reply:
sorry, it seems that @jasonkriss has already pointed this out above.